app/src/main/java/io/heckel/ntfy/msg/NotificationDispatcher.kt

```diff
@@ -19,7 +20,15 @@ class NotificationDispatcher(val context: Context, val repository: Repository) {
     private val distributor = Distributor(context)
 
     fun init() {
-        notifier.createNotificationChannels()
+        notifier.createDefaultNotificationChannels()
+    }
+
+    fun createNotificationChannels(subscription: Subscription) {
+        notifier.createNotificationChannels("" + subscription.id, "" + subscription.id, displayName(subscription))
```

Comment:
One thing that came to my mind this morning: Maybe not the `displayName` should be used here, because it can change after creating the group. In this case the group name must be updated, too, to stay in sync and this must be added to the code.

Maybe the name of the group should always be `server.com/topic`, no matter what the user has entered as a display name? Then it can never change.

Reply:
I think either one is fine. Lazy is fine. :-D

Reply:
Lazy might confuse the user. I can imagine getting a bunch of GitHub issues/Discord messages: "I enabled custom notification sounds for my 'Alerts' subscription, but all I see in the notification channels is "ntfy.sh/alerts-topic-123". Is this the same thing?"

Reply:
> I can imagine getting a bunch of GitHub issues/Discord messages

This is actually a good and major point. Not about the display name (_people know their topic, they have to POST to it_), but about the UX being awkward. It's not clear from the UX what people need to do after toggling the switch in the UI, i.e. that they need to go to the channel settings to configure the rest. 

I suppose we do have the "Channel settings" pref in the UI, but it's not clear that that's related, I think.

Reply:
I added the channel settings prev in the UI so you can test it out, see https://github.com/binwiederhier/ntfy-android/pull/48/commits/b63e5e8bfc5ce117fe1ac4d70b32a3fedc8f8073 - doesn't feel too bad.

<img width="427" alt="Bildschirm­foto 2022-12-06 um 21 54 33" src="https://user-images.githubusercontent.com/148003/206021156-6b6c5c4f-998d-4753-9756-b1fef185292a.png">

I couldn't find out how to scroll to the correct group though, currently it simply opens the notification settings of the app.

---

Regarding the displayName I changed it to the topic short url for now (lazy version): c18662dd1a100288ff2fd3a20b64c51607006e6b

I still can be changed changed to use the `displayName` (with a watcher to change it on display name change), but at least it is consistent for now.